script.js

```diff
@@ -19,3 +19,221 @@ document.addEventListener('click', (e) => {
+  divv.id = index;
+  if (index === 1 || index === 3) {
+    divv.classList.add('reverse');
+  }
+  divv.innerHTML = `
+  <div class="mobile">
+    <img src=${project.image_mobile} alt="" />
+  </div>
+  <div class="desktop"><img src=${project.image_desktop} alt="" /></div>
+  <div class="right">
+    <h2>${project.name}</h2>
+    <div class="back">
+      <div>CANOPY</div>
+      <div class="circle"></div>
+      <div>Back End Dev</div>
+      <div class="circle"></div>
+      <div>2015</div>
+    </div>
+    <p>
+    ${project.description}
+    </p>
+    <ul>
+      <li>HTML</li>
+      <li>CSS</li>
+      <li>JavaScript</li>
+    </ul>
+    <div id='see-project'><a href="">See Project</a></div>
+  </div>
+`;
+
+  works.appendChild(divv);
+});
+
+const boddy = document.querySelector('body');
+
+document.addEventListener('click', (e) => {
+  const target = e.target.closest('#see-project');
+
+  if (target) {
+    e.preventDefault();
+
+    const idd = target.parentNode.parentNode.id;
+
+    const popup = document.createElement('div');
+    popup.className = 'pop';
+    popup.classList.add('tonic');
+    popup.innerHTML = `
+
+<div class="right mobile">
+  <div class="close" >
+   <h2>${projects[idd].name}</h2>
+   <i id="close_pop" class="fa-solid fa-x"></i>
+  
+   
+  </div>
+  <div class="back">
+    <div>CANOPY</div>
+    <div class="circle"></div>
+    <div>Back End Dev</div>
+    <div class="circle"></div>
+    <div>2015</div>
+  </div>
+  <div class="mobile"><img src=${projects[idd].image_mobile} alt="" /></div>
+  <p>
+  ${projects[idd].details}
+  </p>
+  <ul>
+    <li>HTML</li>
+    <li>CSS</li>
+    <li>JavaScript</li>
+  </ul>
+  <div class="see">
+   <a class='host' href=${projects[idd].live}>
+    <span>see live</span>
+    <img src="./images/live.svg"/>
+   </a>
+   <a class='git'href=${projects[idd].source}>
+    <span>see source</span>
+    <i class="fa fa-github" aria-hidden="true"></i>
+   </a>
+
+  </div>
+  
+</div>
+
+
+
+<div class="right desktop">
+  <div class="close" >
+   <h2>${projects[idd].name}</h2>
+   <i id="close_pop" class="fa-solid fa-x"></i>
+   
+  </div>
+  <div class="back">
+    <div>CANOPY</div>
+    <div class="circle"></div>
+    <div>Back End Dev</div>
+    <div class="circle"></div>
+    <div>2015</div>
+  </div>
+  <div class="desktop-image"><img src=${projects[idd].image_desktop} alt="" /></div>
+
+
+  <div class=details>
+   
+  <div class='left'>
+  <p>
+  ${projects[idd].details}
+  </p>
+  </div>
+  <div class='right'>
+  <ul >
+  
+     <li>HTML</li>
+     <li>CSS</li>
+     <li>JavaScript</li>
+   
+  </ul>
+
+  <ul class='ruby' >
+  <li>Ruby</li>
+  <li>Bootstrap</li>
+ </ul>
+  <div class="see">
+  <a class='host' href=${projects[idd].live}>
+    <span>see live</span>
+    <img src="./images/live.svg"/>
+   </a>
+   <a class='git' href=${projects[idd].source}>
+    <span>see source</span>
+    <i class="fa fa-github" aria-hidden="true"></i>
+   </a>
+    </div>
+   </div>
+  </div>
+</div>
```

Comment:
Nice work on adding the functionality of the popup😉 your project looks great👍. But let's try to make the popup responsive so it can match the given[ design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1-468&t=bGWYHZTgBDNQNxPS-0) for mobile👍as it is a requirement that our popup is responsive as it matches the[ design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1-468&t=bGWYHZTgBDNQNxPS-0)😉.

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img width="300" alt="your popup" src="https://user-images.githubusercontent.com/64643995/231216086-67595329-e3a4-4c48-82a8-7c4d71031f9e.png">|<img width="300" alt="design" src="https://user-images.githubusercontent.com/64643995/231215967-20e34b3c-612a-40ae-9a41-d3625681b125.png">|